pkgs/gcloud/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## 0.8.16-wip
```

Comment:
Should we mark this with a breaking version change?
We are still not 1.0.0 so I think we shouldn't worry too much

Reply:
I don't know, it's not likely to break anyone.